src/client.js

```diff
@@ -1032,6 +1035,17 @@ class Client {
         this.clearOptionCache();
     }
 
+    startRefreshCaches() {
```

Comment:
All public functions should have jsdoc and a comment explaining what the function does. 
this looks a good place to pass options to the cache refresher, such as the timer interval